frontend/src/views/Pages/Login.spec.js

```diff
@@ -62,12 +77,33 @@ describe('Login', () => {
       expect(wrapper.find('div.login-form').exists()).toBeTruthy()
     })
 
+    it('calls the communityInfo', () => {
```

Comment:
`commits the community info to the store`. I miss a test that the API was called before.

Reply:
```suggestion
    it('commits the community info to the store', () => {
```